docs/concepts/glossary.md

```diff
@@ -202,7 +202,11 @@ A Peer ID is how each unique IPFS node is identified on the network. The Peer ID
 
 ### Pinning
 
-Pinning is the method of telling an IPFS node that particular data is important and so it will never be removed from that node's cache. [More about Pinning](https://docs.ipfs.io/concepts/persistence/)
+Pinning is the method of telling an IPFS node that particular data is important and so it will never be removed from that node's cache. To learn more: understand [persistence, permanence, and pinning](/concepts/persistence/), then see how to [add local pin](/how-to/pin-files/), and read [what remote pins are](#remote-pinning).
```

Comment:
```suggestion
Pinning is the method of telling an IPFS node that particular data is important and so it will never be removed from that node's cache. To learn more, start by understanding [persistence, permanence, and pinning](/concepts/persistence/); then, see how to [add local pin](/how-to/pin-files/) and read [what remote pins are](#remote-pinning).
```